tests/integration/payment-gateway/SV_WC_Payment_Gateway_Payment_Tokens_Handler_Test.php

```diff
@@ -26,6 +26,27 @@ protected function _after() {
 	}
 
 
+	/**
+	 * @see Framework\SV_WC_Payment_Gateway_Payment_Tokens_Handler::update_tokens()
+	 */
+	public function test_update_tokens() {
```

Comment:
shouldn't we add tests about how data changes when it's updated? currently this only tests that data is retrieved, but there's no insight about its contents

Reply:
I figured any tests for ensuring different properties/meta are set should be testing the `::save()` method specifically. In the update tokens context, we're just testing that it was saved